lib/src/utils/exclude_utils.dart

```diff
@@ -15,3 +15,7 @@ Iterable<Glob> prepareExcludes(
         .map((exclude) =>
             Glob(p.normalize(p.join(root, exclude)).replaceAll(r'\', '/')))
         .toList();
+
+Iterable<String> extractExcludes(String excludePath) => excludePath.isNotEmpty
```

Comment:
@incendial can be simple

=> excludePath.split(',').map((path) => path.trim());